data_transfer/schemas/record.py

```diff
@@ -10,11 +10,14 @@ class Record(BaseModel):
     """
     id: Optional[Any] = Field(alias='_id')
     filename: str
+    device_type: str
     device_id: str
     patient_id: str
     start_wear: datetime
     end_wear: datetime
 
+    vttsma_dump_date: Optional[str]
+
```

Comment:
Could this be `start_wear` and `end_wear` instead, since it stores similar same info? Should we be batching the SMA data per Dump Date rather than wear time (since dump date is within a wear time?)

Reply:
To add to this (having just thought about it), wouldn't the `Record` be the same if the patient has recorded data for two weeks in a row? So having the `end_wear` distinct would be a good way to determine between these Records

I suppose we could create multiple records with the same PatientID/DeviceID/WearPeriod, but do we want `Record` to be unique?

Reply:
Hmm good point, but currently the data within a dump date went beyond the dump dates itself. So I am not sure if that is a strong certainty we can rely on.

Also, as you noticed, VTT doesn't export date regularly, and perhaps we should not rely on that. We could, however, ignore all past 'dumps' if we store that somewhere. Then, we compute with all new dumps (most likely just one) and then this variable isn't needed anymore.

(I'm going to change the 'dump date' to 'export date' - keeps sounding silly as is)

Reply:
I'm adding this to the Issue for separation and clarity. #23